core/src/main/kotlin/com/ritense/valtimo/camunda/health/IncidentHealthIndicator.kt

```diff
@@ -0,0 +1,15 @@
+package com.ritense.valtimo.camunda.health
+
+import org.camunda.bpm.engine.RuntimeService
+import org.springframework.boot.actuate.health.AbstractHealthIndicator
+import org.springframework.boot.actuate.health.Health
+import org.springframework.boot.actuate.health.Status
+
+class IncidentHealthIndicator (
+    private val runtimeService: RuntimeService
+) : AbstractHealthIndicator() {
+    override fun doHealthCheck(builder: Health.Builder?) {
```

Comment:
The builder isn't nullable

```suggestion
    override fun doHealthCheck(builder: Health.Builder) {
```